### PR TITLE
Add piff

### PR DESCRIPTION
**Pull request by jmeyers314**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jmeyers314** [comment]
@conda-forge-admin please rerender

**Comment by beckermr** [comment]
You only need to add piff. The conda package will know to install its deps.

**Comment by jmeyers314** [comment]
> You only need to add piff. The conda package will know to install its deps.

Wasn't sure what the convention was.  Fixed.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "rubin-env" %}
-{% set version = "0.3.1" %}
+{% set version = "0.3.2" %}
```
Comment (timj) [inline recipe/meta.yaml:2]: @ktlim did we decide that adding a new package should be a minor version increment?

Comment (ktlim) [reply recipe/meta.yaml:2]: Yes, we did.  This should be 0.4.0.

Comment (jmeyers314) [reply recipe/meta.yaml]: That was just a wild guess on my part.  Fixed.

**Review by timj — APPROVED** [review]
I'll leave it to @ktlim to merge when he feels the time is right.

**Pull request merged**